api/handlers/process_test.go

```diff
@@ -454,4 +460,79 @@ var _ = Describe("Process", func() {
 			})
 		})
 	})
+
+	Describe("DELETE /v3/processes/:guid/instances/:instance", func() {
```

Comment:
We could restructure this `Describe` in the spirit of the apps handler one

- Start with no instance an no process
- When there is an instance
  - When there is a process